examples/misc/raptor_worker.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, cfg):
 
     # --------------------------------------------------------------------------
     #
+    # TODO: This does not appear to be used.
```

Comment:
It is used [here](https://github.com/radical-cybertools/radical.pilot/blob/devel/examples/misc/raptor.py#L164)